### PR TITLE
Update locale_en-US.ini

### PR DESCRIPTION
**Pull request by ibigbug**
Fixed translation in /admin/config page:

The template is reading {{.Domain}} https://github.com/go-gitea/gitea/blob/a161c2ca5834e950f35fa7546b027f825d4bb5eb/templates/admin/config.tmpl#L20

And the translation shouldn't be showing "SSH Server Domain"

### CODE REVIEW

**Review by zeripath — APPROVED** [review]

**Comment by zeripath** [comment]
Please update the helper too.

File: options/locale/locale_en-US.ini [hunk 1]
```diff
@@ -149,7 +149,7 @@ lfs_path = Git LFS Root Path
 lfs_path_helper = Files tracked by Git LFS will be stored in this directory. Leave empty to disable.
 run_user = Run As Username
 run_user_helper = Enter the operating system username that Gitea runs as. Note that this user must have access to the repository root path.
-domain = SSH Server Domain
+domain = Server Domain
 domain_helper = Domain or host address for SSH clone URLs.
```
Comment (zeripath) [inline options/locale/locale_en-US.ini]: Please update this too.

Comment (ibigbug) [reply options/locale/locale_en-US.ini]: can't see it's referenced: https://github.com/go-gitea/gitea/search?p=1&q=domain_helper

maybe remove the helper?

Comment (lunny) [reply options/locale/locale_en-US.ini]: https://github.com/go-gitea/gitea/blob/main/templates/install.tmpl#L119

Comment (lunny) [reply options/locale/locale_en-US.ini]: Otherwise LGTM

Comment (ibigbug) [reply options/locale/locale_en-US.ini]: updated others also

**Review by lunny — APPROVED** [review]

**Pull request merged**